app/assets/stylesheets/users.scss

```diff
@@ -0,0 +1,7 @@
+.user_registration_form {
+  float: left;
+}
+
+.np_registration_form {
+  float: left;
+}
```

Comment:
@ianroberts131 this is very minor feedback - favor using hyphens over underscores as separators when naming CSS classes. The hyphens are a more widely adopted convention (although underscores work fine).